app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :likes, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
```

Comment:
- [ ] To use the foreign_key: 'author_id' we should define the column author_id in the corresponding migration, like this 👇🏽 

```ruby
# frozen_string_literal: true

class AddUserRefToComment < ActiveRecord::Migration[7.0]
  def change
    add_reference :comments, :author, column: :author_id, null: false, foreign_key: { to_table: :users }
  end
end
```